src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,16 @@
+const ArticleCard = ({article}) => {
+      return (    <section className="art_cards_main">
+                            <li>
+                            <h4>Topic:{article.topic}</h4>
+                            <p>Title :{article.title}</p>
+                                    <p>Author: {article.author}</p>
+                                    <p>Created at:{article.created_at.split('T')[0]}</p>
```

Comment:
I see what you have done to format the date. To make it even nicer, have a look the Date constructor or even a library like moment.js 😄